<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="selenium19" failures="1" tests="1" name="Default test" time="12.511" errors="0" timestamp="2024-12-26T09:39:47 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.857">
    <failure type="java.lang.AssertionError" message="expected [You logged into a secure area!] but found [You logged into a secure area!
×]">
      <![CDATA[java.lang.AssertionError: expected [You logged into a secure area!] but found [You logged into a secure area!
×]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at stepDefs.UserLoginStepDef.should_display_home_page(UserLoginStepDef.java:75)
at ✽.Should display home page(file:///C:/workspace/Monika/BDDPOMCucumber/src/test/resources/feature/UserLogin.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
